__tests__/components/test.js

```diff
@@ -148,21 +148,27 @@ describe('Components', () => {
     const box = wrap.find('.lightbox').at(0);
 
     img.simulate('click');
+    // eslint-disable-next-line jest-dom/prefer-to-have-attribute
     expect(box.getDOMNode(0).hasAttribute('open')).toBe(true);
```

Comment:
Does `.toHaveAttribute('open')` not work with `.getDOMNode()`?

Reply:
It did not. :shrug: